src/administration/web.md

```diff
@@ -57,23 +65,33 @@ During a `merge`:
 
 Rebuilding the application is not necessary if the same code was already built (for any environment): in this case you will see the message ``Slug already built for this tree id, skipping``.
 
-------------------------------------------------------------------------
+After clicking `Merge` a dialog box will appear that will provide commands to execute future merges from the command line using the [Platform.sh CLI](/gettingstarted/cli.md).
+
+![Merge confirmation cli](/images/header_merge_box.png)
 
-![Sync](/images/icon-sync.png)
+#### Sync
 
-### Sync
+![Header Sync](/images/header_sync.png)
 
 Synchronizing means updating the child environment with the code and/or data of its parent and redeploying the child environment.
 
+After clicking `Sync` a dialog box will appear that will provide commands to execute future merges from the command line using the [Platform.sh CLI](/gettingstarted/cli.md).
+
+![sync confirmation cli](/images/header_sync_box.png)
+
 Note that `sync` is only available if your branch has no unmerged commits, and can be fast-forwarded. On `sync`, your code branch will be fast-forwarded to its parent's tip, and the data (e.g. databases) of the services on the branch will be overwritten with an exact copy of the parent's. Syncing of data and code can be done individually, if desired, so you can benefit from having only code changes applied.
```

Comment:
The new text is using `Sync`.  This is showing `sync`.  The capitalization should be consistent.

Reply:
Also, I'm not sure if the "can be fast-forwarded" is true.  You can do 3 way merges.  It wouldn't be very useful if not, no?

Verify this with Engineering, but I think this paragraph is completely wrong.